UnityProject/Assets/Scripts/Tilemaps/Behaviours/Objects/RegisterPlayer.cs

```diff
@@ -104,6 +104,33 @@ public override bool IsPassableFromOutside(Vector3Int from, bool isServer, GameO
 		return IsPassable(isServer);
 	}
 
+	[Command]
+	public void CmdToggleRest(bool layingDown)
```

Comment:
if this is a toggle, why do we also have to tell it if we are laying down or not?

Reply:
also the name can be a little misleading, since it is more about changing the standing/laying down position than it is resting.